### ISSUE TITLE
 Do you use Subdivisions in the learning phase?

### ISSUE THREAD

**Issue by heefe92** [experiencor/keras-yolo3#14]
I want to learn COCO Data.
DarkNet use subdivision because of batchnorm. Is it implemented here?
thank you..

**Comment by experiencor** [comment]
@heefe92 nope

**Comment by heefe92** [comment]
@experiencor Do you have any plane to make?

**Comment by experiencor** [comment]
@heefe92 unfortunately no.

**Comment by heefe92** [comment]
@experiencor Could you tell me why?

**Comment by experiencor** [comment]
@heefe92 I don't know how to do it.
